right-for-life_back/app.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const morgan = require('morgan');
+const bodyParser = require('body-parser');
+const animalRoutes = require('./routes/animalsRoutes.js');
+const newsRoutes = require('./routes/newsRoutes.js');
+const happyStoriesRoutes = require('./routes/happyStoriesRoutes.js');
+const suppliesRoutes = require('./routes/supliesRoutes.js');
+const aboutRoutes = require('./routes/aboutRoutes.js');
+const donateRoutes = require('./routes/donatesRoutes.js');
```

Comment:
Could you please also add Model and Route for EmergencyHelpPage and SingleEmergencyHelpPage?

Reply:
could you pls provide json-example (or maybe some initial data) for them?

Reply:
JSON can be found at https://alex-boklag.github.io/SSA-Demo-AnimalShelter/db/emergency.json

Reply:
added